src/coreclr/nativeaot/System.Private.CoreLib/src/System/Diagnostics/StackFrame.NativeAot.cs

```diff
@@ -163,15 +163,14 @@ internal void AppendToStackTrace(StringBuilder builder)
             {
                 // Passing a default string for "at" in case SR.UsingResourceKeys() is true
                 // as this is a special case and we don't want to have "Word_At" on stack traces.
-                string word_At = SR.GetResourceString(nameof(SR.Word_At), defaultString: "at");
+                string word_At = "at"; //SR.GetResourceString(nameof(SR.Word_At), defaultString: "at");
```

Comment:
This pattern might be fixable by exposing UsingResourceKeys from the SR class and using that info to choose between `SR.Word_At` and `"at"`.